flame/tools/utils.py

```
@@ -4,10 +4,19 @@
 # This source code is licensed under the BSD-style license found in the
 # LICENSE file in the root directory of this source tree.
 
+import torch.nn.functional as F
```

Ruff (0.8.2)

`torch.nn.functional` imported but unused

Remove unused import: `torch.nn.functional`

(F401)